corehq/apps/users/decorators.py

```diff
@@ -30,6 +30,16 @@ def _inner(request, domain, *args, **kwargs):
     return decorator
 
 
+def get_permission_name(permission):
```

Comment:
nitpicky but is a bit odd to define this in `decorators` instead of some permission utils module since it's not a decorator and not used directly by the decorators